src/DesktopTests/UIAutomation/TreeWalkers/TreeWalkerForTestTests.cs

```diff
@@ -31,13 +32,38 @@ public void BeforeEachTest()
         }
 
         [TestMethod]
-        [Timeout(1000)]
         public void RefreshTreeData_ThrowsWhenCancelled()
         {
             var cancellationToken = new CancellationTokenSource();
             cancellationToken.Cancel();
             var dataContext = new DesktopDataContext(Registrar.GetDefaultInstance(), A11yAutomation.GetDefaultInstance(), cancellationToken.Token);
             Assert.ThrowsException<OperationCanceledException>(() => _treeWalker.RefreshTreeData(TreeViewMode.Raw, dataContext));
         }
+
+        [TestMethod]
+        public void RefreshTreeData_RunsAllRules()
+        {
+            _elementMock.Setup(e => e.Framework).Returns(FrameworkId.WPF);
+            var dataContext = new DesktopDataContext(Registrar.GetDefaultInstance(), A11yAutomation.GetDefaultInstance(), CancellationToken.None);
```

Comment:
If there a reason not to use the `DefaultContext` ?
```suggestion
            var dataContext = DesktopDataContext.DefaultContext;
```